sdk/cosmosdb/cosmos/src/request/request.ts

```diff
@@ -68,6 +68,9 @@ export async function getHeaders({
     ...defaultHeaders,
   };
 
+  // Adding SDKSupportedCapabilities header to hint that SDK supports partition merge
+  headers[Constants.HttpHeaders.SDKSupportedCapabilities] = 1;
```

Comment:
use a constant here instead of directly using the value.

Reply:
Done